### ISSUE TITLE
Regarding depth loss implemented in the code

### ISSUE THREAD

**Issue by shreyask3107** [cyhsu14/OmniNeRF#3]
In the following lines of [run_nerf.py](https://github.com/cyhsu14/OmniNeRF/blob/main/run_nerf.py):
```
        if args.use_depth:
            depth_loss = torch.abs(dep - target_d).mean()

```
isn't dep a rendered depth map and target_d from the direction map? Could you please clarify this?
